spec/controllers/relationships_controller_spec.rb

```diff
@@ -0,0 +1,97 @@
+require 'spec_helper'
+
+describe RelationshipsController do
+  describe "GET index" do
+    let(:alice) { Fabricate(:user) }
+    let(:bob) { Fabricate(:user) }
+
+    it_behaves_like "users must be signed in" do
+      let(:action) { get :index }
+    end
+
+    it "assigns @relationships to the current users relationships" do
+      set_current_user(alice)
+      Fabricate(:relationship, follower_id: alice.id, leader_id: bob.id)
```

Comment:
Your `Relationship` model has `follower` and `leader` methods, you should use them instead of `_id`.

`Fabricate(:relationship, follower: alice, leader: bob)`